js/implementation.native.ts

```diff
@@ -44,13 +44,13 @@ let updatesEnabled = false;
  */
 export function setRNConfiguration(config: GeolocationConfiguration) {
   RNCGeolocation.setConfiguration({
-    ...config,
+    skipPermissionRequests: config.skipPermissionRequests,
     authorizationLevel:
-      config?.authorizationLevel === 'auto'
+      config.authorizationLevel === 'auto'
```

Comment:
`config` is not undefined, only the property, but this check is safe for `config.authorizationLevel === undefined`.